media-sound/mixxx/mixxx-9999.ebuild

```diff
@@ -78,13 +78,13 @@ DEPEND="
 "
 
 PATCHES=(
-	"${FILESDIR}"/${PN}-2.0.0-docs.patch
+	"${FILESDIR}"/${PN}-9999-docs.patch
```

Comment:
Please don't use a patch for this. Use the below sed line.

Reply:
Should be fixed in c9b2107